### DISCUSSION TITLE
Help configuring ffmpeg settings to output to detector

### DISCUSSION THREAD

**Discussion by msettlemeyer** [blakeblackshear/frigate#9185] [Ask A Question]
Hi, I'm having a hard time understanding how I should be adding parameters to double the volume off the input stream for use in an audio detector. Can anyone help point me in the right direction?

**Comment by NickM-27** [comment]
there's no direct way to do this since the output args for the audio detector are not configurable. Does the camera not support volume control?

If not then you'd need to use go2rtc to do this

**Comment by msettlemeyer** [comment]
Ok, thanks. I probably can. I think my cam has goRtc. I'm just not familiar with it yet. Audio detection seems to be working great. Thanks for your hard work.
